realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -226,9 +228,8 @@ protected void removeHandler() {
      * the last transaction was committed.
      *
      * @param destination file to save the Realm to.
-     * @throws java.io.IOException if any write operation fails.
      */
-    public void writeCopyTo(File destination) throws java.io.IOException {
+    public void writeCopyTo(File destination) {
```

Comment:
The the biggest change is `Realm.getInstance()` will throw a `IOExcpetion` and user has to catch it which doesn't seem to be necessary at all...